src/ol_infrastructure/applications/ocw_site/Pulumi.yaml

```diff
@@ -0,0 +1,5 @@
+name: ol-infrastructure-ocw_site-application
```

Comment:
For the name we should stick with `-` over `_`.

Reply:
I think this got filled in by the `copier` command. I'd put in an underscore because the other project directories under `applications` have underscores. Is there some way around having it fill it in that way and keep consistency with the other directory names?

Reply:
Yeah, that's fine, I just meant that we should edit the name attribute in the yaml. Not really significant, just a nit pick.

Reply:
Sure, that's fine. I've updated the `name` property.